presto-product-tests/src/main/java/io/prestosql/tests/cli/PrestoCliLauncher.java

```diff
@@ -78,9 +78,18 @@ protected void launchPrestoCli(List<String> arguments)
 
     protected ProcessBuilder getProcessBuilder(List<String> arguments)
     {
-        return new ProcessBuilder(ImmutableList.<String>builder()
+        ImmutableList<String> command = ImmutableList.<String>builder()
                 .add(JAVA_BIN, "-Xmx50m", "-cp", CLASSPATH, Presto.class.getCanonicalName())
                 .addAll(arguments)
-                .build());
+                .build();
+
+        String commandText = Joiner.on(' ').join(command);
+
+        return new ProcessBuilder(ImmutableList.of("bash", "-c", commandText));
```

Comment:
Why do we need bash for that? Can't we just execut java?

Reply:
Without bash you would not need any escaping.

Reply:
To be able to redirect stdin to it

Reply:
`java.lang.ProcessBuilder#redirectInput()` seems to support that?

Reply:
Did we arrive at a conclusion here?

Reply:
I've removed bash